src/metabase/cmd/copy.clj

```diff
@@ -104,15 +104,14 @@
 (def ^:private chunk-size 100)
 
 (defn- insert-chunk!
-  "Insert of `chunk` of rows into the target database table with `table-name`."
+  "Insert of `chunkk` of rows into the target database table with `table-name`."
   [target-db-type target-db-conn table-name chunkk]
-  (print (color/blue \.))
-  (flush)
+  (log/debugf "Inserting chunk of %d rows" (count chunkk))
```

Comment:
I thought about adding some useful info like `(count chunkk)`, but wouldn't that realize the entire seq at this point?

Then again, that's probably fine once we get to this point. I don't feel strongly about it either way.

Reply:
The chunk is only at most 100 rows, and we have to realize the whole thing because we insert all 100 rows at once in the code above. So it won't make a difference here. That's why we're chunking it tho, so we don't have more than 100 in memory at once